test/tools/Dockerfile

```diff
@@ -45,7 +45,7 @@ COPY --from=installer /aws-cli-bin/ /usr/local/bin/
 RUN aws --version
 
 # Install eksctl
```

Comment:
Why do we need this?

Reply:
because I tried to test this on my arm instance :D

Reply:
We are running this on ARM instances?

Reply:
Don't need to block on this, just curious

Reply:
we're not running this on ARM instances, but wanted to test it and the instance I was using happened to be ARM. Trivial change to make it ARM compatible so figured I'd PR it. We may want to run it on ARM instances at some point